spec/processSpec.ts

```diff
@@ -9,7 +9,7 @@ function removeCR(f: string): string {
 }
 
 function read(fileName: string) {
-   return fs.readFileSync(fileName).toString();
+   return fs.readFileSync(fileName).toString().replace(/\r\n/g, "\n");
```

Comment:
Normalize string representation of files because line endings are different between windows and mac.